index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a verifiable digital credential (VDC).
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, this document includes a registry of credential presentation exchange protocols to ensure interoperability.
+    </p>
+    <p>
+      The API is designed to support the following key properties:
+      <ul>
+        <li>
+          By separating the act of requesting from the specific protocol, we can enable flexibility and adaptability in both the protocol and credential formats. This way, the pace of changes in browsers won't hinder progress or block new developments.
+        </li>
+        <li>
+          Require request transparency, enabling user-agent inspection for risk analysis.
+        </li>
+        <li>
+          Assume response opacity (encrypted responses), enabling verifiers and holders to control where potentially sensitive PII is exposed.
+        </li>
+        <li>
+          Websites cannot access any information without user interaction. This API ensures that sites cannot silently query for digital credentials or communicate with wallet providers without the user's active participation and confirmation of each action.
+        </li>
+      </ul>
+    </p>
     <p id="credential-type-examples">
-      There are many types of digital credential that can be presented using
+      There are many types of verifiable digital credentials that can be presented using
```

Comment:
```suggestion
      There are many types of verifiable digital credential that can be presented using
```

Reply:
@TallTed can you explain this one? Singular sounds very weird when read aloud. If I were to replace VDCs with "documents", just to try a different word, I don't think "document" would be correct ("...there are many types of document that can be presented using...")

Reply:
Sorry for delay. 

`"...there are many types of document that can be presented using..."` would be correct.

`"...there are many types of documents that can be presented using..."` would not.

Try changing `types` to `kinds` and `document` to `pizza`. I expect "There are many kinds of `pizzas` that you can order" and "You can order many kinds of `pizzas`" will both sound odd to you. I expect "There are many kinds of `pizza` that you can order" and "You can order many kinds of `pizza`" will both sound right.

If you really want the plural `credentials`, I might suggest `Verifiable digital credentials of many types can be presented...`